sdk/core/azure-core-management/src/test/java/com/azure/core/management/implementation/polling/LROPollerTests.java

```diff
@@ -474,7 +474,7 @@ public String getName() {
             } catch (InterruptedException e) {
                 //
             }
-            Assertions.assertEquals(count, getCallCount.get());
+            //Assertions.assertEquals(count, getCallCount.get());
```

Comment:
Any reason for commenting this out? Looks like the poller tests should not be impacted by this PR.

Reply:
These have been fixed and merged in after the commit this branch was checked out from. Cherry picked to make it so CI fails less.